ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java

```diff
@@ -122,6 +157,9 @@ private static void usage() {
     private final long maxInterval;
     private final String schemaRegistryUrl;
     private final InputStream propertiesFile;
+    private final int numThreads;
```

Comment:
Let's add these into the `usage()` string above, so users will know we've added awesome new features :)